catalystwan/models/configuration/feature_profile/sdwan/policy_object/application_list.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+from typing import List, Union
+
+from pydantic import BaseModel, Field, PrivateAttr
+
+from catalystwan.api.configuration_groups.parcel import Global, _ParcelBase
+from catalystwan.models.configuration.feature_profile.sdwan.policy_object.object_list_type import PolicyObjectListType
+
+
+class ApplicationType(str, Enum):
```

Comment:
Looks like it is not statically defined list, the available options are provided by vManage:
`/dataservice/app-registry/applications`
Please remove that enum.

Reply:
deleted